index.src.html

```diff
@@ -340,6 +340,13 @@ <h3 id="concept-reports">Reports</h3>
   the value of the <code>User-Agent</code> <a>header</a> of the <a>request</a>
   from which the report was generated.
 
+  Note: We collect the <code>User-Agent</code> header for each report, even
+  though we can also get this header from the report upload requests.  This
```

Comment:
To me this wording is a little confusing - suggesting they're the same thing ("also get ..."). How about something roughly like:

> The User-Agent header in the report represents the User-Agent sent by the browser for the page which generated the report. This is potentially distinct from the User-Agent sent in the HTTP headers for the report in case where the browser has chosen to use a non-default User-Agent string such as "request desktop site" feature.

Reply:
Done